Neuro/Minigames/Solvers/StabilizeSteeringSkeldSolver.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using Neuro.Cursor;
+using UnityEngine;
+
+namespace Neuro.Minigames.Solvers;
+
+[MinigameSolver(typeof(NavigationMinigame))]
+public class StabilizeSteeringSkeldSolver : MinigameSolver<NavigationMinigame>
+{
+    protected override IEnumerator CompleteMinigame(NavigationMinigame minigame, NormalPlayerTask task)
+    {
+        InGameCursor.Instance.SnapToCenter();
```

Comment:
To make this look better, I would suggest doing SnapTo the starting position of the cursor, then startholding, then CoMoveTo center.

Reply:
This way the movement looks like sliding and not snapping